test/message.test.js

```diff
@@ -131,7 +131,7 @@ describe("Message", () => {
 
 			beforeEach(() => {
 				options = {
-					status: 'success',
+					state: 'success',
```

Comment:
how did this one not fail before?

Reply:
Oh I did miss that one. That particular test is not asserting anything about the state and o-message doesn't error given an invalid state (it seems). 

Looks like `contruct-element.test.js` is testing states correctly though.